db/migrate/20250608013722_create_active_operator_operations.rb

```diff
@@ -0,0 +1,15 @@
+class CreateActiveOperatorOperations < ActiveRecord::Migration[8.0]
+  def change
+    create_table :active_operator_operations do |t|
+      t.references :record, polymorphic: true, null: false
```

Comment:
We probably want to make sure we have an index on this:

```suggestion
      t.references :record, polymorphic: true, null: false, index: true
```